### PR TITLE
docs: add links to top with jump2header

### PR DESCRIPTION
**Pull request by strdr4605**
Added links to the table of contents for better navigation.

[Demo](https://github.com/strdr4605/takes/blob/jump2header/README.md#quick-start)

Generated with:

```
jump2header --start "Quick Start" --slug "contents"
```

If you don't like the PR but interested in links to top see [jump2header](https://github.com/strdr4605/jump2header#readme) for more options.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#1072](https://github.com/yegor256/takes/issues/1072) is assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), here is [why](https://www.0crat.com/footprint/C3FFK3YAY/93f8415a-7dbf-4ede-8f9e-dc193a9f1513); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by fabriciofx** [comment]
@strdr4605 it seems fine to me. @paulodamaso can you merge it, please?

**Comment by fabriciofx** [comment]
@strdr4605 just for curiosity: is is possible change the link icon (the arrow)?

**Comment by strdr4605** [comment]
@fabriciofx Yes. At the moment 5 icons are available:
image

**Comment by fabriciofx** [comment]
@paulodamaso which them do you prefer?

File: README.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ This [blog post](http://www.yegor256.com/2015/03/22/takes-java-web-framework.htm
 - [How to contribute](#how-to-contribute)
 - [Got questions?](#got-questions)
 
-## Quick Start
+## Quick Start[⬆](#contents)<!-- Link generated with jump2header -->
```
Comment (andreoss) [inline README.md:93]: @strdr4605 Do we have to keep these comments?

**Comment by andreoss** [comment]
@paulodamaso It will look like that.  To me, it looks a bit odd, but the feature itself is nice

Screenshot from 2020-09-17 08-30-40

**Review by andreoss — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (strdr4605) [reply README.md:93]: @andreoss No, it's there only for info purpose for others to understand.

**Comment by strdr4605** [comment]
> @paulodamaso It will look like that. To me, it looks a bit odd, but the feature itself is nice
> 
> Screenshot from 2020-09-17 08-30-40

This is how it looks on my machine (macOS).

image

**Review by andreoss — APPROVED** [review]
@paulodamaso Looks okay now.

**Comment by paulodamaso** [comment]
@strdr4605 Thank you for the PR, but I wouldn't like to have a dependency for generating the headers for the README. Also, I think that these images are a little cumbersome, I'd prefer a thinner approach, like just `^` for example

**Comment by strdr4605** [comment]
@paulodamaso 

**jump2header** is not a dependency. You can use it one time to generate links for 30+ sections, then all new sections and links can be added manually.

It's not possible to add links as text in headers as it will break the anchor (only emojis). 
I suggest placing links bellow the header or at the end of the section. 
In this case, we can use custom text instead of emoji. [Example](https://github.com/strdr4605/jump2header/blob/master/examples/text.md)

**Comment by andreoss** [comment]
@paulodamaso I think we can merge that.

**Comment by paulodamaso** [comment]
@strdr4605 I don't want that every contributor that edit the readm.md file must execute this script, so that's why I'm closing this issue

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @andreoss[/z](https://www.0crat.com/u/andreoss), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
Code review was too long (28 days), architects (@paulodamaso) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Comment by 0crat** [comment]
Pull request [#1072](https://github.com/yegor256/takes/issues/1072) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

**Comment by sereshqua** [comment]
@0crat quality bad

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31)

**Pull request closed without merging**